bin/cache.ml

```diff
@@ -35,8 +35,16 @@ let trim ~trimmed_size ~size =
   with
   | Error s -> User_error.raise [ Pp.text s ]
   | Ok { trimmed_bytes } ->
+    let pp_bytes trimmed_bytes =
```

Comment:
I think this code should live somewhere in Stdune as it's generally useful.

Reply:
(In fact, don't we already have something similar?)

Reply:
I couldn't find anything about formatting bytes like this in the repo. I can add it to stdune, though I am not sure who else will use it. Where should I include it?

Reply:
There's `bytes_unit` in `Decoder` (`dune_sexp`). You might be able to reuse the table.

Reply:
Ha, indeed! Thanks @emillon. I remember seeing something like this -- it even has a comment from me on top :)

Reply:
I've factored the parsing and printing of byte units through a library in stdune called Bytes_units. This is backwards compatible and allows aliases for units to easily be added.

Reply:
Thanks!